### ISSUE TITLE
Question - Can a core use a config set and have its own schema file?

### ISSUE THREAD

**Issue by Martin-Andersen** [SolrNet/SolrNet#408]
First sorry for spamming, but I am desperate. And after seeing you pluralsigt courses I think that you can answer my question.

I am upgrading from Solr 4.3 to latest version of Solr. Today I have a cluster running on AWS in SolrCloud mode where many of the cores share the same files but each have their own schema. The sharing are done with copying the file with a script and we are not storing the file in zookeeper, so no auto scaling.

I am trying to use a config set for the common files and have a schema file for each core. Can it be done? So far I have no luck, getting an error:

` org.apache.solr.common.SolrException:org.apache.solr.common.SolrException: Could not load conf for core entry_dada: Error loading both non-managed schema 'managed-schema-dada' and managed schema 'managed-schema'` 

Have also tried without conf directory and just putting the schema file beside the core.propperties file.

So in short how should I organise config sets and cores if I want to reuse common files and have all the configuration stored in zookeeper?

**Comment by xavier-morera** [comment]
Hi @Martin-Andersen IMHO: why are you not storing the common files in each collection? Is it because of size constraints? 

Whenever we have several cores, each one has their own set of config files and schema. 

Also I had a situation where I had several cores that each one handled a particular customer, instead of copying stuff I simply created a separate request handler for each customer so I had a common schema, and each application that called Solr simply had a configuration to use a different request handler. In the handler I specified an invariants so that results narrowed down to a particular customer. 

Hope this helps.

**Comment by Martin-Andersen** [comment]
Thanks for answering. 

No size constraints, I just thought the I could use a config set for the common files. As a coder I hate duplication (-:

How should a folder structure look like if you want to upload it to zookeeper?
cores/core#1/core.properties
cores/core#1/conf/schema.xml
cores/core#1/conf/solrconfig.xml
cores/core#1/conf/lang
cores/core#1/conf/other files
cores/core#2 .....

Are paths in properties on the core.properties file relativ to the folder? e.g schema=schema-core#1.xml

I dont want Solr to guess the types of fields when I index my documents but I would also like to have the freedom to define a new field in solr admin, is that possible?

OK that was a lot of question, sorry for that. But real world examples on setting up Solr are more rare then 2 Sundays in a week.

I would really like to gain a deeper knowledge about Solr. Can you recommend any books, blogs or online training for Solr?  

Thanks
